### ISSUE TITLE
Color (theme) customization?

### ISSUE THREAD

**Issue by zhiyuanzhai** [daylen/stockfish-mac#72]
It would be better if we can customize the color of the chess board.

**Comment by 0xferit** [comment]
Closing as a duplicate of #24, which asks for the same thing and is still open.

The request itself is live, not rejected: board colours are still hardcoded in `SFMBoardView.m`, there are no colour keys in `SFMUserDefaults`, and Preferences has no colour well. It is simply tracked in the older issue.

Recording one thing here so it is not lost with the thread: this was asked again in 2021, five years after #24 was filed, by a different person.
